<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/pics/WhatsApp Image 2023-12-18 at 22.27.06_64ed168a.jpg</normaloff>:/pics/WhatsApp Image 2023-12-18 at 22.27.06_64ed168a.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="deletee">
        <property name="toolTip">
         <string>Delete List</string>
        </property>
        <property name="toolTipDuration">
         <number>-5</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/pics/WhatsApp Image 2023-12-18 at 22.27.05_2149ed7a.jpg</normaloff>:/pics/WhatsApp Image 2023-12-18 at 22.27.05_2149ed7a.jpg</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="toolTip">
         <string>Clear Marked As Done</string>
        </property>
        <property name="toolTipDuration">
         <number>-5</number>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="toolTip">
         <string>Mark All As Done </string>
        </property>
        <property name="toolTipDuration">
         <number>-5</number>
        </property>
        <property name="text">
         <string>Mark All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="toolTip">
         <string>Search Bar</string>
        </property>
        <property name="toolTipDuration">
         <number>-5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="toolTip">
         <string>Search</string>
        </property>
        <property name="toolTipDuration">
         <number>-5</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/pics/WhatsApp Image 2023-12-18 at 22.27.04_7644e5b4.jpg</normaloff>:/pics/WhatsApp Image 2023-12-18 at 22.27.04_7644e5b4.jpg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>CLOSE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">background color: rgb(52, 52, 52)</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>627</width>
         <height>392</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="toolTip">
         <string>Add</string>
        </property>
        <property name="toolTipDuration">
         <number>-5</number>
        </property>
        <property name="styleSheet">
         <string notr="true">background color: rgb(52, 52, 52)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/pics/WhatsApp Image 2023-12-18 at 22.27.05_c8dc785c.jpg</normaloff>:/pics/WhatsApp Image 2023-12-18 at 22.27.05_c8dc785c.jpg</iconset>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">font: bold;
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="toolTip">
         <string>Priority</string>
        </property>
        <property name="toolTipDuration">
         <number>-5</number>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold</string>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="tabletTracking">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Date &amp; Time</string>
        </property>
        <property name="toolTipDuration">
         <number>-5</number>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="displayFormat">
         <string>d/M/yyyy h:mm AP</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
